core-charter.txt

```diff
@@ -1,40 +1,65 @@
 
 CoAP supports various forms of "caching". Beyond the benefits of caching already well known from REST, caching can be used to increase energy savings of low-power nodes by allowing them to be normally-off [RFC 7228]. For example, a temperature sensor might wake up every five minutes and send the current temperature to a proxy that has expressed interest in notifications. When the proxy receives a request over CoAP or HTTP for that temperature resource, it can respond with the last notified value, instead of trying to query the Device which may not be reachable at this time. The CoRE WG will continue to evolve this model to increase its practical applicability.
 
-The CoRE WG will perform maintenance as well as possible updating and complementing on its first four standards-track specifications as required:
+The CoRE WG has completed a set of standards-track specifications building on the CoAP specification [RFC 7252] and will perform maintenance as well as possible updating and complementing the framework in areas of resource discovery, group communication, data formats, operation and management, and CoAP related security.
 
-- RFC 6690
-- RFC 7252
-- RFC 7641
-- RFC 7959
+Candidate work items include:
 
-The CoRE WG will continue to evolve the support for CoAP group communication originally developed in the Experimental RFC 7390. The CoRE WG will not develop a solution for reliable multicast delivery of CoAP messages, which will remain Non-Confirmable when sent over multicast.
+* evolve the support for CoAP group communication originally developed in the Experimental RFC 7390(but not develop a solution for reliable multicast delivery of CoAP messages)
 
-RFC 7252 defines a basic HTTP mapping for CoAP, which was further elaborated in RFC 8075. This mapping will be evolved and supported by further documents as required.
+* continue defining transport mappings for alternative transports as required, both IP-based and non IP-based 
 
-CoAP was initially designed to work over UDP and DTLS. Later on, mapping were defined between CoAP and IP-based transports, especially TCP and WebSockets including a secure version over TLS (RFC 8323). The CoRE WG will continue defining transport mappings for alternative transports as required, both IP-based and non IP-based (e.g., SMS and Bluetooth), working with the Security Area on potentially addressing the security gap. This includes defining appropriate URI schemes. Continued compatibility with CoAP over SMS as defined in OMA Lightweight Machine-to-Machine (LwM2M) will be considered. Furthermore, the CoRE WG will work on solutions to facilitate the signaling of transports available to a Device.
+* solutions to facilitate the signalling of transports available to a Device.
```

Comment:
I find this objective very vague.  It sounds like an entirely new GRASP or mDNS or DHCP based protocol.

Reply:
That's not what it is.  We probably have to expand this text slightly.